Sources/RichTextKit/Context/RichTextContext.swift

```diff
@@ -19,6 +20,8 @@ import SwiftUI
  */
 public class RichTextContext: ObservableObject {
 
+    var userActionPublisher: PassthroughSubject<RichTextUserAction, Never> = .init()
```

Comment:
This should be a `let` property - so that someone does not change it inadvertently.